hw/top_earlgrey/doc/dv_plan/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Earlgrey Chip DV Plan"
+---
+
+## Goals
+* **DV**
+  * Verify `top_earlgrey` features by running dynamic simulations with a SV/UVM based testbench.
+  * Verify the integration of all pre-verified IPs instantiated in the chip.
+  * Verify the integration and the internal design of un-verified IPs instantiated in the chip.
+  * Verify system level scenarios for correctness of our design assumptions and behavior.
+  * Verify the full chip configuration and memory address space by running the automated tests.
+  * Stress test the XBAR structures in the chip.
+* **FPV**
+  * Verify the connectivity of signals (that are excluded from functional DV above) from block A to block B.
+  * Secure verification - TBD
```

Comment:
I'm not sure any of our secure FPV tools could ever work at full chip, but perhaps there is a heavy-black-box way to do it?

Reply:
Yeah, sure. I added a line for this.